pom.xml

```diff
@@ -447,6 +447,12 @@
 				<type>jar</type>
 			</dependency>
 			<dependency>
+		    	<groupId>com.tdunning</groupId>
+		    	<artifactId>json</artifactId>
+				<version>1.2-SNAPSHOT</version>
```

Comment:
I think the problem is that the new library escapes special characters also for JSONFunction, so 'alert("")' becomes 'alert(\"\")' and can not be evaluated by javascript.